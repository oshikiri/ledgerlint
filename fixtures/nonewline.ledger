2020-03-26 * super market
  Expences:Household essentials  200 JPY
  Assets:Cash
  ; some comment
2020-03-26 * super market
  Expences:Household essentials  200 JPY
  Assets:Cash
