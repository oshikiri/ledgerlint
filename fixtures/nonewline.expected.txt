fixtures/nonewline.ledger:1 parsePostingStr is failed: '2020-03-26 * super market'
